### ISSUE TITLE
Question: I210 controller 802.1Qav support

### ISSUE THREAD

**Issue by VinodTadvai** [AVnu/Open-AVB#198]
Is there software in openAVB that utilizes 802.1Qav support on I210 ethernet controller card?

**Comment by intel-ethernet** [comment]
I believe the simple_talker example application demonstrates how to configure the Qav shaper on the I210 adapter.

ekm

**Comment by grandcat** [comment]
Yes. Have a closer look at the function `igb_set_class_bandwidth()` in the talker code.
